### ISSUE TITLE
Trouble with using 'assess.yr' for documents that span multiple calendar years

### ISSUE THREAD

**Issue by grinnellm** [cgrandin/csas-latex#118]
I just noticed an error caused by using the variable `assess.yr` in the text. We started working on our herring assessment in Sept 2017, and just sent the final draft to CSAS in Jan 2018. Note that 
`assess.yr <- as.numeric(substr(Sys.Date(), 1, 4))`
which changed our 'assessment year' from 2017 to 2018 in some of our figure and table captions, and document text. Perhaps this is more of a warning to others rather than an issue.

**Comment by cgrandin** [comment]
Yeah we changed this for hake so that it's a hardwired year. I got carried away and put that code in at some point!!
